web/viewer.css

```
@@ -1430,6 +1345,144 @@ dialog :link {
   z-index: 50000; /* should be higher than anything else in PDF.js! */
 }
 
+#findbar {
+  background-color: var(--toolbar-bg-color);
+  cursor: default;
+  font: message-box;
+  font-size: 12px;
+  height: auto;
+  inset-inline-start: 64px;
+  line-height: 14px;
+  margin: 4px 2px;
+  min-width: 300px;
+  padding: 0 4px;
+  position: absolute;
+  text-align: left;
+  top: var(--toolbar-height);
+  z-index: 30000;
+
+  * {
+    float: var(--inline-start);
+    position: relative;
+  }
+
+  > div {
+    height: var(--toolbar-height);
+  }
+
+  button {
+    font: message-box;
+    outline: none;
+  }
+
+  input {
+    font: message-box;
+    outline: none;
+
+    &:focus-visible + label {
```

A general question, that might apply in more places below, is if we could avoid some unnecessary repetition in the rules (as done in the old code)?
```suggestion
  :is(button, input) {
    font: message-box;
    outline: none;
  }

  input {
    &:focus-visible + label {
```

---

Yes, I was a bit on the fence before on which approach I liked better, but having thought about it more I tend to agree that combining them is better overall. I think I have updated all occurrences now; thanks!

---

Thanks for doing this, since I think that we should be (at least somewhat) mindful about total filesize of the CSS and by combining stuff we can avoid it increasing too much.